models/errors_whitebox_test.go

```diff
@@ -0,0 +1,30 @@
+package models
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSimpleError_Error(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	e := simpleError{message: "foo"}
+	assert.Equal(t, "foo", e.Error())
+}
+
+func TestBadParameterError_Error(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	e := BadParameterError{parameter: "foo", value: "bar"}
+	assert.Equal(t, fmt.Sprintf(stBadParameterErrorMsg, e.parameter, e.value), e.Error())
+}
+
+func TestNotFoundError_Error(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.UnitTest)
+	e := NotFoundError{entity: "foo", ID: "bar"}
+	assert.Equal(t, fmt.Sprintf(stNotFoundErrorMsg, e.entity, e.ID), e.Error())
```

Comment:
> Not quite sure what you're testing here...

@aslakknutsen, I test the [`Error()`](https://github.com/almighty/almighty-core/blob/44af5a728cb6c8fb41e1ebd7269ee7226bc52338/models/errors.go#L51) method of the `NotFoundError` struct.

Those tests seemed like low handing fruits.